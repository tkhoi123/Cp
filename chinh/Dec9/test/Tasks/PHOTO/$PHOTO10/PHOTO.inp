97
65 66 71 61 16 79 84 18 97 24 69 41 94 7 14 8 87 64 95 88 34 33 75 96 92 70 6 23 83 42 47 81 77 80 68 28 32 37 22 57 90 29 60 55 59 40 26 13 93 89 4 36 5 86 91 51 49 35 20 50 27 73 72 74 63 19 44 30 12 21 46 52 85 1 43 10 45 25 39 76 15 2 17 11 48 9 3 31 38 53 78 54 82 62 58 67 56 
66 4 71 16 79 90 14 11 84 97 24 41 94 7 69 18 87 64 95 57 34 33 8 92 70 6 23 42 47 81 9 77 80 72 68 28 32 37 61 22 29 60 55 59 27 40 26 96 13 93 89 30 36 76 56 65 91 51 49 38 35 86 50 73 74 19 21 75 46 52 44 85 1 5 43 10 45 25 88 39 15 2 17 48 63 12 31 20 83 53 78 54 82 3 62 58 67 
65 55 66 4 71 79 11 84 46 49 18 97 41 94 14 69 87 64 95 57 34 33 8 80 92 73 75 70 6 83 47 81 77 1 28 32 16 40 37 7 61 22 90 54 60 59 89 27 26 96 13 15 23 93 36 76 53 56 5 86 91 35 20 50 72 74 25 19 30 12 21 52 44 68 85 43 31 10 45 88 39 2 17 48 9 3 63 38 24 78 29 82 62 42 58 67 51 
65 66 4 71 16 11 18 97 24 41 7 14 69 87 64 95 57 34 33 8 92 75 70 6 23 83 42 81 94 80 68 28 37 61 90 39 29 55 22 2 79 59 43 27 40 26 96 13 93 89 76 56 5 86 91 47 51 49 35 20 50 73 72 84 74 19 30 12 21 46 52 44 85 1 10 45 32 25 88 15 48 9 3 63 31 38 77 53 78 54 82 17 36 60 62 58 67 
65 66 4 16 71 79 11 84 18 24 41 94 7 14 37 69 57 34 8 13 95 81 78 92 62 75 33 6 23 83 42 47 77 21 80 68 32 61 22 90 29 60 55 27 40 26 96 93 89 36 48 76 56 85 5 70 86 82 91 51 49 20 50 73 72 74 87 64 19 30 12 46 52 44 1 43 97 10 45 35 25 59 88 28 39 15 2 17 9 3 63 31 38 58 53 54 67 
