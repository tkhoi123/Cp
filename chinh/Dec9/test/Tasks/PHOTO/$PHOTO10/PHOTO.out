65 66 4 71 16 79 11 84 18 97 24 41 94 7 14 69 87 64 95 57 34 33 8 92 75 70 6 23 83 42 47 81 77 80 68 28 32 37 61 22 90 29 60 55 59 27 40 26 96 13 93 89 36 76 56 5 86 91 51 49 35 20 50 73 72 74 19 30 12 21 46 52 44 85 1 43 10 45 25 88 39 15 2 17 48 9 3 63 31 38 53 78 54 82 62 58 67 